docs/embedded/getting-started/spembedded-for-vscode.md

```diff
@@ -31,10 +31,17 @@ If you don't already have a tenant, you can get a tenant through the [Microsoft
 ![Install](../images/vsx-images/n2vsx-signin.png)
 
 - Authentication opens a new tab in an external browser to grant permissions
+
+![auth allow extension to open uri](../images/vsx-images/auth-allow-extension-uri.png)
```

Comment:
Use full words "authorization" vs "auth"

Reply:
Sorry, I got confused. This is for the VSCode extension itself. This permission may indeed be required. I thought this was for their own client that was created